app/code/Magento/Catalog/Block/Product/View.php

```diff
@@ -63,6 +63,11 @@ class View extends AbstractProduct implements \Magento\Framework\DataObject\Iden
     protected $productRepository;
 
     /**
+     * @var \Magento\Catalog\Model\ProductFactory
+     */
+    protected $_productFactory;
```

Comment:
please don't introduce protected access modifiers, use private one instead.
Also don't use _ prefix for attribute names, as this violates PSR naming